config/locales/en.yml

```diff
@@ -143,6 +146,9 @@ en:
         pass_crypt: "Password"
         pass_crypt_confirmation: "Confirm Password"
     help:
+      doorkeeper/application:
+        confidential: "Application will be used where the client secret can be kept confidential, eg mobile apps and single page apps"
```

Comment:
I suspect you moved this and then rebased on a newer version of my tree where I had edited the original although your version doesn't look exactly like the original upstream version either as it seems to have the logic backwards?

Reply:
Oops, I didn't notice the change in the text. Good catch. I'll update this shortly.